src/Native/Bootstrap/main.cpp

```diff
@@ -348,3 +376,14 @@ int main(int argc, char* argv[])
 
     return retval;
 }
+#endif // !CORERT_DLL
+
+#ifdef CORERT_DLL
+static struct InitializeRuntimePointerHelper
+{
+	InitializeRuntimePointerHelper()
```

Comment:
The indentation is off here. Tabs vs. spaces?

Reply:
Spaces 😄